kadasrouting/valhalla/client.py

```diff
@@ -27,7 +27,7 @@ def __init__(self, connector=None):
     def isAvailable(self):
         return self.connector.isAvailable()
 
-    def route(self, qgspoints, profile, avoid_polygons, options):
+    def route(self, qgspoints, profile, avoid_polygons, options, patrol_polygons=[]):
```

Comment:
Using a list as a default parameter can introduce a gotcha. See https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
I can put None, anyway the test is `if patrol_polygons` which goes both with an empty list or `None`

Reply:
Yes, None is better in this case. Thanks!